src/redux/reducers/dialogsReducer.js

```diff
@@ -1,4 +1,5 @@
-import { UPDATE_NEW_MESSAGE_BODY, SEND_MESSAGE } from './actions/actionTypes';
+import { UPDATE_NEW_MESSAGE_BODY, SEND_MESSAGE } from '../actions/actionTypes';
+import { NEXT_ID } from '../../utils/constants';
```

Comment:
Consider moving this import.
I already asked your to reorganize the order of imports several times.

Reply:
/fxd